src/Test_Rewriting/Tests/Tests.cs

```diff
@@ -0,0 +1,29 @@
+namespace Tests;
+using MyQueue;

```

Comment:
```suggestion
namespace Tests;

using MyQueue;
```
И название не очень. Лучше PriorityQueue.Tests